ext/web/02_event.js

```diff
@@ -482,7 +482,7 @@ function getRoot(eventTarget) {
 function isNode(
   eventTarget,
 ) {
-  return Boolean(eventTarget && ReflectHas(eventTarget, "nodeType"));
+  return eventTarget && eventTarget.nodeType !== undefined;
```

Comment:
if `eventTarget` is falsy, this may return non-boolean value.

```suggestion
  return eventTarget?.nodeType !== undefined;
```